experimental/addons/envoy-gateway/README.md

```diff
@@ -0,0 +1,3 @@
+# envoy-gateway
+
+Envoy Gateway is an open source project for managing Envoy Proxy as a standalone or Kubernetes-based application gateway.
```

Comment:
Can you help add some usage docs here? Users need the basic knowledge about how to use this addon.

And please also add some application examples.

Reply:
Thanks, I will address this comment ASAP.